multivendor-frontend/src/pages/product.tsx

```
@@ -0,0 +1,394 @@
+  Typography,
+  Drawer, // Import Drawer
+} from "@mui/material";
+import { GrFilter } from "react-icons/gr";
+import { ArrowDropDown, GridOn, Star, ViewList } from "@mui/icons-material";
+import { BsSortUpAlt } from "react-icons/bs";
+
+const SideFilterProduct = React.lazy(
+  () => import("./components/SideFilterProduct")
+);
+
+const Product = () => {
```

Biome

[error] Shouldn't redeclare 'Product'. Consider to delete it or rename it.

'Product' is defined here:


(lint/suspicious/noRedeclare)